src/components/CountryContent/index.js

```diff
@@ -61,6 +60,11 @@ const useStyles = makeStyles(theme => ({
     fontFamily: theme.typography.body1.fontFamily,
     paddingTop: '1.125rem',
     paddingBottom: '1.0625rem'
+  },
+  marginBottom: {
```

Comment:
This spaces the component too much:
![s](https://user-images.githubusercontent.com/1779590/73726591-e5606400-4740-11ea-983f-fdd340465e9d.png)

And we need to left align labels in mobile